procedures/foreman/branch.md.erb

```diff
@@ -81,12 +81,7 @@
   - Add a Katello version mapping
   - Make any changes if needed (usually happens when there are tag changes)
   - Deploy using `scp collection-mash-split.py root@koji.katello.org:/usr/local/bin`
-- [ ] Add the new release to the JJB job definitions in [jenkins-jobs](https://github.com/theforeman/jenkins-jobs)
-  - [ ] Create a `<%= release %>.groovy` in `pipelines/vars/foreman/` describing the operating systems the release supports
-  - [ ] Add the release to the list in `theforeman.org/yaml/views/release.yml`
-  - [ ] Add `<%= release %>` to the version list in `theforeman.org/yaml/includes/foreman_versions.yaml.inc`
-  - [ ] Add `<%= release %>` to the version list in `centos.org/jobs/foreman-pipelines.yml`
-  - [ ] Create `test_<%= release.tr('.', '_') %>_stable.yaml` and `test_proxy_<%= release.tr('.', '_') %>_stable.yaml` in `yaml/jobs/`
+- [ ] Branch in [jenkins-jobs](https://github.com/theforeman/jenkins-jobs) `./branch-foreman <%= release %> KATELLO_VERSION`
```

Comment:
`And open a PR` ? Given how much we call out to `Open PR` in other places I think it is good to be explicit here unless the intent is to branch and direct push (eww?).